cli/src/commands/build_docker_scraper.py

```diff
@@ -8,13 +8,34 @@ def get_name(self):
     def get_description(self):
         return 'Build scraper images (dev, prod)'
 
+    def get_options(self):
+        return [{"name": "test",
+                 "description": "build the test image",
+                 'optional': False}]
+
+    @staticmethod
+    def test_parse(args):
+        if len(args) < 2:
+            return False
+        if isinstance(args[1], bool):
+            return args[1]
+        return isinstance(args[1], str) and args[1].lower() == 'true'
+
     def run(self, args):
-        py3 = self.get_option('python3', args)
+        
+        test = self.get_option('test', args)
+
+        if test:
+            return self.build_docker_file("scraper/dev/docker/Dockerfile.test",
+                                          "algolia/docsearch-scraper-test")
+
+        code = self.build_docker_file("scraper/dev/docker/Dockerfile.base",
+                                      "algolia/base-docsearch-scraper")
```

Comment:
Shouldn't we have both images follow the same pattern in naming? Like `docsearch-scraper-test` and `docsearch-scraper-base` instead of `base-docsearch-scraper`?

Reply:
Good idea. I will do it